tests/jballerina-unit-test/src/test/resources/test-src/imports/PredeclaredImportsTestProject/predeclared-modules.bal

```diff
@@ -53,9 +53,14 @@ function testTrxInfoRecordTypeUse() returns boolean {
     return true;
 }
 
+function testCallInFunction()returns any|error {
```

Comment:
```suggestion
function testCallInFunction() returns any|error {
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/36700